Sources/Wrap.swift

```diff
@@ -127,6 +154,10 @@ public protocol WrapCustomizable {
      *  implementation of the `keyForWrapping(propertyNamed:)` method.
      */
     var wrapKeyStyle: WrapKeyStyle { get }
+    /**
+     * The style that wrap should ignore nil values or explicitly put JSON null value
+     */
+    var wrapNilStyle: WrapNilStyle { get }
```

Comment:
However, using bool value provides easier integration but  using enum makes possible to add more options in future